language_names/scraper-sil.py

```diff
@@ -15,25 +17,24 @@
     parser.add_argument('-t', '--table', help='language names table', default='languageNames')
     args = parser.parse_args()
 
-    print('BEGIN TRANSACTION;')
-
-    sil_tsv = urllib.request.urlopen(sil_names).read().decode('utf-8')  # Python's csv module chokes badly on this
+    request = urllib.request.Request(sil_names, headers=user_agent)
+    sil_tsv = urllib.request.urlopen(request).read().decode('utf-8')  # Python's csv module chokes badly on this
     tsv_contents = sil_tsv.splitlines()
-    for tsv_line in tsv_contents[1:]:  # skip the header
-        matches = tsv_format.match(tsv_line)
 
-        if matches:
-            iso639_3 = matches.group('Id')
-            iso639_1 = matches.group('Part1')
-            iso639 = iso639_1 or iso639_3
+    with open('language_names/scraped-sil.tsv', 'a') as f:
```

Comment:
Why are we opening as append instead of write?

Reply:
I am writing the data of this file to scraped-sil.tsv. Should I remove the old data I received from scraped-sil.sql? I didn't want to delete that data so I opened it in append

Reply:
Append makes no sense. Then these files would just keep growing... In theory, the scraped files should only be improving so it's fine to overwrite them.

Reply:
Okay I will change that